### ISSUE TITLE
Tags not applied properly on test cases outside of describe block

### ISSUE THREAD

**Issue by nathaner** [testomatio/app#639]
**Describe the bug**

In a test file, if some tests are inside a describe block and some others are outside, tags are not applied properly. The test outside the describe block will have the tags of the latest describe block applied to it, erasing its own tags.

**To Reproduce**
1. Create a file `sandbox.spec.ts` with the following content:
```ts
import test from '@playwright/test'

test.describe('first test suite @first', () => {
  test('test case 1', () => {
    const a = 1
  })
  test('test case 2', () => {
    const a = 1
  })
})

test.describe('second test suite @second', () => {
  test('test case 3', () => {
    const a = 1
  })
  test('test case 4', () => {
    const a = 1
  })
})

test('test case 5 @sanity', () => {
  const a = 1
})
```
3. Import the test to Testomat:
```sh
TESTOMATIO=<API_KEY> npx check-tests@latest Playwright "**/*{.,_}{test,spec}.ts" --typescript 
```
4. You can see the last test case (`test case 5`) doesn't have the tag `@sanity` but only the tag `@second` as you can see in the screenshot. The tag `@sanity` does appear on the global Tests page though. 

**Expected behavior**

Tags should be applied on a parent-child relationship, independently of whether the test case is in the same file or not.

**Screenshots**
![image](https://user-images.githubusercontent.com/25883088/204870457-58e6a873-399a-467b-9359-39e7983f7370.png)
![image](https://user-images.githubusercontent.com/25883088/204870508-04157688-bd31-4c2e-8154-da3f054b8011.png)
![image](https://user-images.githubusercontent.com/25883088/204870537-77633b13-4f1c-4205-8336-df738ca88406.png)

**Desktop (please complete the following information):**
 - OS: Ubuntu
 - Browser chrome
 - Browser Version 107.0.5304.121 (Official Build) (64-bit)
 - Application: production 

**Additional context**
Issue was reproduced using Playwright. I don't know how this behaves on other frameworks.

**Comment by TetianaKhomenko** [comment]
Hi @nathaner! Can I ask you to check this issue on your side, please? The issue should be fixed on check-tests 0.8.2. Thank you in advance.

**Comment by TetianaKhomenko** [comment]
closing till next occurrence
